README.md

```diff
@@ -113,12 +113,22 @@ Shutting down...
 
 ## Using Docker
 
+You can simply run Docker within the VM. However, if you install the Docker Engine on the host, you can use Docker commands natively on the host Mac. Install the Docker Engine either using the instructions at https://docs.docker.com/installation/mac/
```

Comment:
Sorry for this comment again.
It is not Docker Engine for Mac but just a client for the Docker Engine (daemon), precisely.
You can use `Docker client` instead of `Docker Engine` for it.